src/SignalR/clients/csharp/Client/src/Microsoft.AspNetCore.SignalR.Client.csproj

```diff
@@ -11,4 +11,7 @@
     <Reference Include="Microsoft.AspNetCore.Http.Connections.Client" />
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.SignalR.Client.Tests" />
```

Comment:
Wait this is a thing now? You don't need to put this in assembly attributes 😱

Reply:
It IS a thing! You still need to have the full public key for things that aren't built by the repo, but if it's built here you can just use the simple assembly name and the rest is automated!